test-files/cases/reference/unifying/highway/highway-3829/Expected.osm

```diff
@@ -766,8 +766,7 @@
         <tag k="highway" v="tertiary"/>
         <tag k="horse" v="no"/>
```

Comment:
This one lost a couple of tags. Doesn't seem like a big deal. Not sure why, though.